hbase-server/src/test/java/org/apache/hadoop/hbase/backup/TestHFileArchiving.java

```diff
@@ -185,11 +215,30 @@ private void testArchiveStoreFilesDifferentFileSystems(String walDir, String exp
     when(mockedFile.getPath()).thenReturn(filePath);
     when(mockedFileSystem.rename(any(),any())).thenReturn(true);
     archivingFunction.apply(conf, mockedFileSystem, mockedRegion, tableDir, family, list);
-    ArgumentCaptor<Path> pathCaptor = ArgumentCaptor.forClass(Path.class);
-    verify(mockedFileSystem, times(2)).rename(pathCaptor.capture(), any());
```

Comment:
note that we used to validate that:
1. we had two rename calls (one to sideline the timestamped backup, and one to actually do the archive)
2. we only validated the first call - that the source file was in the archive folder... which is not the important part we want to validate here.